lib/builtin.t

```diff
@@ -51,6 +51,17 @@ package FetchStoreCounter {
     is(prototype(\&builtin::is_bool), '$', 'is_bool prototype');
 }
 
+# float contants
+{
+    use builtin qw( inf nan );
+
+    ok(inf, 'inf is true');
+    ok(inf > 1E10, 'inf is bigger than 1E10');
+    ok(inf == inf, 'inf is equal to inf');
```

Comment:
A good test here would be inf == inf + 1 or similar

Reply:
`inf == inf / 2` might be a reasonable test.

Reply:
There are already comprehensive tests for handling of inf and nan values in `t/op/infnan.t`

Reply:
Added `inf == inf + 1`. There's no need to go overboard on them; most of the value tests are indeed elsewhere. This is just testing that the `builtin::inf` function appears to have grabbed the correct value there.

Reply:
$x == $x + 1 will also be true for large (not infinite) floats.